inc/core/styles/css_prop.php

```diff
@@ -334,24 +334,33 @@ public static function transform_directional_prop( $meta, $device, $value, $css_
 
 		if ( count( array_unique( $filtered ) ) === 2 && $value['top'] === $value['bottom'] && $value['right'] === $value['left'] ) {
 
+			if ( self::has_calc( $value['top'] )  &&  self::has_calc( $value['right'] ) ) {
+				return $css_prop . ':' . $value['top'] . ' ' . $value['right'] . ';';
+			}
+
```

Comment:
Not necessarily, just a suggestion:

I wonder if we can follow up a different strategy here instead of having a `has_calc` method. I would like to suggest;

adding $args param to the `\Neve\Core\Styles\Css_Prop::transform_directional_prop` method for general usage, then pass a flag in $args from https://github.com/Codeinwp/neve/blob/1506666ed30ca4af92faf44930ac20a20675f1fb/inc/core/styles/css_vars.php#L204-L205

so, the idea is decide returning top and right values directly according to the method argument instead of `self::has_calc` I'm think this way seems more performant due to I think strpos is a more costly performance-wise

```suggestion
			if ( isset($args['button_paddings'] ) {
				return $css_prop . ':' . $value['top'] . ' ' . $value['right'] . ';';
			}

```

Reply:
Of course that depends to the how many \Neve\Core\Styles\Css_Prop::has_calc will be executed. As I see that method is run dozens.